pybamm/__init__.py

```diff
@@ -163,6 +163,8 @@ def version(formatted=False):
 from .solvers.scipy_solver import ScipySolver
 from .solvers.scikits_dae_solver import ScikitsDaeSolver
 from .solvers.scikits_ode_solver import ScikitsOdeSolver
+from .solvers.scikits_ode_solver import JacobianFunctionCV
```

Comment:
Don't think these functions need to be used outside of scikits_ode_solver, so can probably get rid of this line